app/Http/Controllers/Admin/BatchController.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use Illuminate\Http\Request;
+use App\Http\Controllers\Controller;
+use App\Repositories\BatchRepository;
+use App\Repositories\WorkspaceRepository;
+use App\Repositories\PositionRepository;
+use App\Repositories\ProgramRepository;
+use App\Http\Requests\BatchRequest;
+
+class BatchController extends Controller
+{
+    protected $batch;
+    
+    public function __construct(
+        BatchRepository $batch,
+        ProgramRepository $programRepository,
+        PositionRepository $positionRepository,
+        WorkspaceRepository $workspaceRepository
+    )
+    {
+        $this->batch = $batch;
+        $this->programRepository = $programRepository;
+        $this->positionRepository = $positionRepository;
+        $this->workspaceRepository = $workspaceRepository;
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionClosingBrace.SpacingBeforeClose</br>**Severity**: error</br>**File**: app/Http/Controllers/Admin/BatchController.php</br>**Line**: 29</br>Function closing brace must go on the next line following the body; found 1 blank lines before brace